src/serde_xml_plist.rs

```diff
@@ -106,15 +107,21 @@ mod de {
             Some(ValueKeyword::Data) => {
                 //FIXME: remove this + base64 dep when/if we merge
                 //<https://github.com/ebarnard/rust-plist/pull/122>
-                let b64_str = map.next_value::<&str>()?;
+                // NOTE: Use Cow here because serde needs ownership if it has
```

Comment:
This is a random thing I stumbled over, put https://github.com/linebender/norad/blob/e58e32009786e767066b9af2cba17bcff23c9ae3/src/serde_xml_plist.rs#L490 into the mutator sans Designspace and run the tests, you'll get a weird de error. Googling it yields an issue where someone suggests using Cow. Basically, we didn't catch this before because no Designspace file had a date or data lib thing.